src/utils/GraphHelpers.ts

```diff
@@ -44,3 +44,39 @@ export function prepScopes(...scopes: string[]) {
   };
   return [new AuthenticationHandlerOptions(undefined, authProviderOptions)];
 }
+
+/**
+ * Helper method to chain Middleware when instantiating new Client
+ *
+ * @param {...Middleware[]} middleware
+ * @returns {Middleware}
+ */
+export function chainMiddleware(...middleware: Middleware[]): Middleware {
+  const rootMiddleware = middleware[0];
+  let current = rootMiddleware;
+  for (let i = 1; i < middleware.length; ++i) {
+    const next = middleware[i];
+    if (current.setNext) {
+      current.setNext(next);
+    }
+    current = next;
+  }
+  return rootMiddleware;
+}
+
+/**
+ * converts a blob to base64 encoding
+ *
+ * @param {Blob} blob
+ * @returns {Promise<string>}
+ */
+export function blobToBase64(blob: Blob): Promise<string> {
```

Comment:
Should this move to the generic utils file since it is not specific to graph?

Reply:
It should!